scripts_windows/build_all.bat

```diff
@@ -0,0 +1,19 @@
+@ECHO OFF
+SETLOCAL
+SET EL=0
```

Comment:
What does this variable do?

Reply:
`EL` is a local variable to store the overall "errorlevel" (return value) that we want to return from the script.  Notice that later lines set EL to non-zero when a program that the batchfile calls itself returns a non-zero errorlevel.  (My searching seemed to indicate there is no way to automatically propagate non-zero errorlevels up like in bash's `set -e`.)